src/db.rs

```diff
@@ -1123,6 +1123,17 @@ impl DB {
             );
         }
     }
+
+    pub fn set_option(&self, name: &str, value: &str) -> Result<(), Error> {
```

Comment:
Have you thought about to make this method similar to original function? What if I want to change more than one parameter ? The implementation as I can see use synchronisation to apply this changes in runtime. So this is not a cheap operation.